components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/IdPInitLogoutRequestProcessor.java

```diff
@@ -122,10 +124,26 @@ private void init(QueryParamDTO[] queryParamDTOs) {
 
         for (QueryParamDTO queryParamDTO : queryParamDTOs) {
             if (SAMLSSOConstants.QueryParameter.SP_ENTITY_ID.toString().equals(queryParamDTO.getKey())) {
-                this.spEntityID = queryParamDTO.getValue();
+                String issuer = splitAppendedTenantDomain(queryParamDTO.getValue());
+                String issuerQualifier = getSPQualifierExists(queryParamDTOs);
```

Comment:
Could move 128 to 133 to separate private method "resolveIssuerQualfier", as same logic repeated another place in this PR

Reply:
Added a new method "resolveIssuerQualfier" to SAMLSSOUtil class.